src/components/match-card.tsx

```diff
@@ -111,11 +136,19 @@ export const MatchDetailsCard = memo(
             <div class={matchNumStyle}>{`Match ${matchName.num}`}</div>
           )}
         </div>
-        {match.time && (
-          <time dateTime={match.time.toISOString()}>
-            {formatTime(match.time)}
-          </time>
-        )}
+        <div class={eventAndTimeStyle}>
+          {matchPage && (
+            <div class={eventNameStyle}>
+              {' '}
+              at <a href={`/events/${eventKey}`}>{eventName}</a>{' '}
+            </div>
+          )}
+          {match.time && (
+            <time dateTime={match.time.toISOString()} class={dateTimeStyle}>
+              {formatTime(match.time)}
+            </time>
+          )}
+        </div>
```

Comment:
@varCepheid When I look at a match page, this placement of the link to the event page isn't really intuitive to me:

https://deploy-preview-1522--peregrine.netlify.app/events/2023orwil/matches/qm1

To me it feels like the link would be better suited in the navigation bar instead of in the match information card (especially to keep the match information card more consistent between pages):

![2023-06-10T17:03:55,740243471-07:00](https://github.com/Pigmice2733/peregrine-frontend/assets/13206945/16f393e9-bfc4-4a49-8ead-8c4a827551a8)

Reply:
Having the link in the navigation bar would also be something we can do across pages - for example this page (team@event) should probably have a link to the event in it (which was previously a back button). It could have the event name in the navigation bar link to the event:

https://deploy-preview-1522--peregrine.netlify.app/events/2023orwil/teams/1425?stat=auto::Docked

Same for team-matches@event: https://deploy-preview-1522--peregrine.netlify.app/events/2023orwil/teams/1425/matches

and team-comments@event: https://deploy-preview-1522--peregrine.netlify.app/events/2023orwil/teams/1425/comments

I'm sure there are lots of others I'm not thinking of. I think for any event-specific page, there should be a way to get from that page to the event page.

Reply:
I did this and it works well by what I can see. In addition to the pages you mentioned, I added it to the analysis table page.